R/path-explorer.R

```diff
@@ -47,15 +48,21 @@ nextStep <- function(listSoFar, originalString, arrayToGo) {
 #'
 #' liver_volume_path <- tree$Organism$Liver$Volume$path
 #' @export
-getSimulationTree <- function(simulationOrFilePath) {
+getSimulationTree <- function(simulationOrFilePath, returnParameterTree = FALSE) {

```

Comment:
Would rename the new argument to `parameterPathsOnly` or `parametersOnly`
Because when I see `returnParameterTree = FALSE` I think "return the tree except the parameters"